src/gui/processing/qgsprocessingmeshdatasetwidget.cpp

```diff
@@ -265,6 +265,11 @@ void QgsProcessingMeshDatasetGroupsWidgetWrapper::setMeshLayerWrapperValue( cons
     mWidget->setMeshLayer( meshLayer, layerFromProject );
 }
 
+QStringList QgsProcessingMeshDatasetGroupsWidgetWrapper::compatibleParameterTypes() const
+{
+  return QStringList() << QgsProcessingParameterMeshDatasetGroups::typeName();
```

Comment:
I think this should probably accept numeric and string types too. If a single int value is used as the parameter value, is that acceptable? Accepting numeric/string types here would make the algorithms much more flexible in models.

Also we may consider adding number/string output types to compatibleOutputTypes here too.

Reply:
thanks for the information, I continue to discover the processing model frame work...
I have also added the possibility to set the dataset time with string and QDateTime